torch/optim/lars.py

```diff
@@ -0,0 +1,168 @@
+from .optimizer import Optimizer, required, _use_grad_for_differentiable  # type: ignore[attr-defined]
+import torch
+from typing import List, Optional
+from torch import Tensor
+
+__all__ = ["LARS", "lars"]
+
+
+class LARS(Optimizer):
+    """Implements LARS algorithm."""
+
+    def __init__(
+        self,
+        params,
+        lr=required,
```

Comment:
Nit: Why not set a default value similar to other optimizers?

Reply:
Do you mean to set a default value for the learning rate? I didn't because SGD, Adam, and other optimizers haven't one.